hardhat.config.ts

```diff
@@ -148,6 +163,45 @@ task('collector:withdraw', 'Withdraws funds from a collector contract')
     await withdraw(taskArgs, hre);
   });
 
+task(
+  'collector:addToken',
+  'Allow the collector to receive payments in additional tokens'
+)
+  .addParam('collectorAddress', 'address of the collector we want to modify')
+  .addParam(
+    'tokenAddress',
+    'address of the token we want to allow in the collector'
+  )
+  .setAction(async (taskArgs: AddCollectorTokenArgs, hre) => {
+    await addTokenToCollector(taskArgs, hre);
+  });
+
+task(
+  'collector:getTokens',
+  'Allow the collector to receive payment in additional tokens'
+)
+  .addParam('collectorAddress', 'address of the collector we want to modify')
```

Comment:
```suggestion
  .addParam('collectorAddress', 'address of the collector contract')
```

Reply:
I changed it to: `address of the collector contract to modify`